### PR TITLE
Promisified

### PR DESCRIPTION
**Pull request by SamVerschueren**
Converted the library from callbacks to promises.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -1,12 +1,8 @@
-'use strict';
-var test = require('ava');
-var runApplescript = require('./');
+import test from 'ava';
+import fn from './';
 
-test(function (t) {
-	t.plan(2);
+test(async t => {
+	const res = await fn('return "unicorn"');
```
Comment (arthurvr) [inline test.js]: Is there still a need to create this constant? Might as well simply do this:

``` js
t.is(await fn('return "unicorn"'), 'unicorn')
```

File: package.json [hunk 2]
```diff
@@ -24,7 +24,17 @@
...
+  },
   "devDependencies": {
-    "ava": "0.0.4"
+    "ava": "*",
+    "xo": "*"
+  },
+  "xo": {
+    "ignore": [
+      "test.js"
+	]
```
Comment (arthurvr) [inline package.json]: Indentation

File: package.json [hunk 3]
```diff
@@ -24,7 +24,17 @@
...
+    "pify": "^2.2.0",
+    "pinkie-promise": "^1.0.0"
+  },
   "devDependencies": {
-    "ava": "0.0.4"
+    "ava": "*",
+    "xo": "*"
+  },
+  "xo": {
+    "ignore": [
```
Comment (arthurvr) [inline package.json]: `ignore` or [`ignores`](https://github.com/sindresorhus/xo#ignores)?

File: test.js [hunk 4]
```diff
@@ -1,12 +1,8 @@
...
 
-	runApplescript('return "unicorn"', function (err, res) {
-		t.assert(!err, err);
-		t.assert(res === 'unicorn');
-	});
-});
+	t.is(res, 'unicorn');
+});
```
Comment (arthurvr) [inline test.js]: Nitpick but this misses a final newline :)

**Comment by arthurvr** [comment]
Thanks again! :)

File: package.json (hunk 3 above)
Comment (SamVerschueren) [inline package.json]: It seems they both work. But fixed according to the docs :).

File: test.js (hunk 4 above)
Comment (SamVerschueren) [inline test.js]: Good catch, missed it because XO ignores the file :).

File: test.js (hunk 1 above)
Comment (SamVerschueren) [inline test.js]: Not really, but I tend to do it separately because I think its faster to see what happens :).

Comment (arthurvr) [inline test.js]: I'd prefer the latter as its shorter and IMHO doesn't really change the readability but it's up to @sindresorhus, WDYT?

Comment (sindresorhus) [inline test.js]: Yeah, put it inline.

**Comment by SamVerschueren** [comment]
Thanks for the feedback!

**Pull request closed without merging**